### PR TITLE
🛠️ [Techdebt] Refactor code and inline fees

### PR DESCRIPTION
**Pull request by kcw-grunt**
# Overview
Updated to match the service fees found in Litewallet iOS
Reviewed and updated old code that needed to be removed or updated

### CODE REVIEW

File: app/src/main/java/com/breadwallet/tools/manager/FeeManager.java [hunk 1]
```diff
@@ -69,6 +85,57 @@ public void setFees(long luxuryFee, long regularFee, long economyFee) {
         // TODO: to be implemented when feePerKB API will be ready
     }
 
+    public static void updateFeePerKb(Context app) {
```
Comment (josikie) [inline app/src/main/java/com/breadwallet/tools/manager/FeeManager.java:88]: Cool!

File: app/src/main/java/com/breadwallet/tools/util/Utils.java [hunk 2]
```diff
@@ -255,4 +285,47 @@ public static String fetchPartnerKey(Context app, PartnerNames name) {
...
+        }
+        else {
+             return (long) ((3.00 / doubleRate) * 100_000_000.0);
+        }
+    }
+    private static boolean isBetween(double x, double lower, double upper) {
+        return lower <= x && x <= upper;
+    }
+    public static Set<String> litewalletOpsSet(Context app) {
+        List<String> addressList = Collections.singletonList(Utils.fetchPartnerKey(app, PartnerNames.LITEWALLETOPS));
```
Comment (josikie) [inline app/src/main/java/com/breadwallet/tools/util/Utils.java:323]: what does singletonList do?

**Review by josikie — APPROVED** [review]
Wow, cool! Thank you! @kcw-grunt

File: app/src/main/java/com/breadwallet/tools/util/Utils.java (hunk 2 above)
Comment (kcw-grunt) [reply app/src/main/java/com/breadwallet/tools/util/Utils.java:323]: trying to remember but you can google. Im not a big Java dev...still learning and I used this after research ....;)

File: app/src/main/java/com/breadwallet/presenter/entities/PaymentItem.java [hunk 3]
```diff
@@ -5,25 +5,29 @@ public class PaymentItem {
 
     public byte[] serializedTx;
     public String[] addresses;
+    public String opsAddress;
```
[the remark this answers is not in the document]
Comment (kcw-grunt) [reply app/src/main/java/com/breadwallet/presenter/entities/PaymentItem.java]: this is the destination address

**Pull request merged**